guix.scm

```diff
@@ -0,0 +1,70 @@
+;;
+(use-modules
+ (gnu packages llvm)
+ (gnu packages ncurses)
+ (gnu packages pkg-config)
+ (gnu packages sdl)
+ (guix build-system cmake)
+ (guix gexp)
+ (guix git-download)
+ ((guix licenses) #:prefix license:)
+ (guix packages)
+ (guix utils)
+ (ice-9 regex))
+
+(define-public %tangerine-origin
+  (local-file
+   "." "tangerine-src"
+   #:recursive? #t
+   #:select?
+   (let* ((src-dir (current-source-directory))
+          (checked-in?
+           (or (and src-dir
+                    (git-predicate (canonicalize-path src-dir)))
+               (lambda (path stat)
+                 #t)))
+          (so-dll-rx
+           (make-regexp "\\.(so|dll)$"))
+          (so-or-dll?
+           (lambda (path stat)
+             (regexp-exec so-dll-rx path)))
+          (source-file?
+           (lambda (path stat)
+             (and (checked-in? path stat)
+                  (not (so-or-dll? path stat))))))
+     source-file?)))
+
+(define-public tangerine
+  (package
+    (name "tangerine")
+    (version "0.0")
+    (source %tangerine-origin)
+    (build-system cmake-build-system)
+    (native-inputs
+     (list clang-toolchain
```

Comment:
Commenting out `clang-toolchain` leads to using GCC, which runs into a different error.